src/Utils.js

```diff
@@ -169,7 +170,14 @@ export function coercePropValue(propValue) {
   }
 
   // coerce to boolean
-  return propValue === 'true' ? true : false;
+  if (propValue === 'true') return true;
+  if (propValue === 'false') return false;
+
+  // user provided an unquoted string value
+  throw new Error(
```

Comment:
I also like what's going on this [PR](https://github.com/airbnb/enzyme/pull/110/files#diff-c33afeb87a3143ade6f458848655083bR141) with prefacing these errors with `Enzyme::Selector`